### PR TITLE
make ACTIVE_PILLOW_NAMES unique and sorted

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

File: src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 1]
```diff
@@ -610,13 +610,11 @@ MAPS_LAYERS = json.loads('{{ MAPS_LAYERS|to_json }}')
 PILLOWTOP_MACHINE_ID = '{{ localsettings.PILLOWTOP_MACHINE_ID }}'
 {% endif %}
 
-ACTIVE_PILLOW_NAMES = list(set([
-{% for _, pillows in app_processes_config.pillows.items() %}
-    {% for pillow_name in pillows.keys()|unique %}
-        "{{ pillow_name }}",
-    {% endfor %}
+ACTIVE_PILLOW_NAMES = [
+{% for pillow_name in app_processes_config.values() | map("dict2items") | flatten | map(attribute="key") | unique | sort %}
```
Comment (mkangia) [inline src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2:614]: wow, that is something chaining, were you able to test it?

Comment (snopoke) [reply src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2:614]: Yes I did test it:

vars.yml
```yaml
pillows:
  pillow1:
    case-pillow:
    form-pillow:
  pillow2:
    case-pillow:
    user-pillow:
```

test.py.j2
```python
ACTIVE_PILLOW_NAMES = [
{% for pillow_name in pillows.values() | map("dict2items") | flatten | map(attribute="key") | unique | sort %}
    "{{ pillow_name }}",
{% endfor %}
]
```

```
skelly@~/src/commcare-cloud (ansible) [sk/unique-pillows] $ ansible all -i "localhost," -c local -m template -a "src=test.py.j2 dest=./test.txt" -e @vars.yml; cat test.py
localhost | CHANGED => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    }, 
    "changed": true, 
    "checksum": "2213cf5e2496dfe56a601a129881ee883b53989c", 
    "dest": "./test.txt", 
    "gid": 1000, 
    "group": "skelly", 
    "md5sum": "15b4646970f450434cdef75c91aac5b7", 
    "mode": "0664", 
    "owner": "skelly", 
    "size": 82, 
    "src": "/home/skelly/.ansible/tmp/ansible-tmp-1601539404.63-129308413587130/source", 
    "state": "file", 
    "uid": 1000
}
ACTIVE_PILLOW_NAMES = [
    "case-pillow",
    "form-pillow",
    "user-pillow",
]
```

**Comment by mkangia** [comment]
Thanks @snopoke 
Build failure looks related

```
TASK [commcarehq : copy localsettings] *****************************************

fatal: [localhost]: FAILED! => {"changed": false, "msg": "AnsibleFilterError: dict2items requires a dictionary, got <class 'ansible.parsing.yaml.objects.AnsibleUnicode'> instead."}
```

**Comment by snopoke** [comment]
@mkangia build passing

**Review by mkangia — APPROVED** [review]

**Pull request merged**